src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs

```diff
@@ -28,6 +28,10 @@ internal static void Register(CommandLineApplication app)
                     "-p|--protocol",
                     "Protocol to resolve TagHelperDescriptors with.",
                     CommandOptionType.SingleValue);
+                var outputPath = config.Option(
+                    "-o|--outputPath",
+                    "The projects output path.",
```

Comment:
Got it. So it's going to be `The path to which the output of the project will go. :trollface:`.

Perfect verbage